config/ecs/ecs-cloudwatch.yaml

```diff
@@ -19,47 +19,6 @@ processors:
       - system
       - ecs
       - ec2
-  filter:
```

Comment:
We need to remove line 11 as well. We are not using the `awsecscontainermetrics` anymore.

Reply:
addressed the change with force push.